Lib/ast.py

```diff
@@ -1053,15 +1054,77 @@ def visit_AsyncWith(self, node):
         with self.block(extra=self.get_type_comment(node)):
             self.traverse(node.body)
 
+    def _str_literal_helper(
+        self, value, quote_types=("'", '"', '"""', "'''"), escape=""
+    ):
+        """Helper for writing string literals, minimising escapes.
+        Returns (possible quote types, string literal to write).
+        """
```

Comment:
I don't get the reason for returning a list of quotes instead of the most suitable one. As far as I can see, both usages just accesses the first item so we may just return the most suiting quote
```suggestion
        """Helper for writing string literals, minimising escapes. Returns the
        possible quote type, and the escaped valued."""
```

Reply:
I think somewhere we've agreed on American English spellings, so I'd make that "minimizing".

Reply:
In `visit_JoinedStr` we actually repeatedly filter the list of possibly quote types within the loop, so we can't just return a single quote.

I'll make the American English change.

Reply:
Added a comment to `visit_JoinedStr` to help make that stand out.